src/_tests/fixtures/43136/derived.json

```diff
@@ -6,7 +6,7 @@
   "headCommitAbbrOid": "e686353",
   "headCommitOid": "e6863537248bbfee8f0ef8c636bb00c25cf40b96",
   "lastPushDate": "2020-03-16T14:26:42.000Z",
-  "lastActivityDate": "2020-03-16T14:26:42.000Z",
+  "lastActivityDate": "2020-03-16T14:31:40.000Z",
```

Comment:
This is the correct `lastActivityDate`: these fixtures have `pullRequest.comments` but not `pullRequest.timelineItems(itemTypes: ISSUE_COMMENT)` so using `pullRequest.comments` now results in the correct date.